pkg/file/writer.go

```diff
@@ -917,6 +926,41 @@ func populatePartials(kongState *state.KongState, file *Content) error {
 	return nil
 }
 
+func populateKeys(kongState *state.KongState, file *Content,
+	config WriteConfig,
+) error {
+	keys, err := kongState.Keys.GetAll()
+	if err != nil {
+		return err
+	}
+	for _, k := range keys {
+		k := FKey{Key: k.Key}
+		utils.ZeroOutID(&k, k.Name, config.WithID)
+		utils.ZeroOutTimestamps(&k)
+		file.Keys = append(file.Keys, k)
+	}
+	sort.SliceStable(file.Keys, func(i, j int) bool {
+		return compareOrder(file.Keys[i], file.Keys[j])
+	})
+	return nil
+}
+
+func populateKeySets(kongState *state.KongState, file *Content) error {
+	sets, err := kongState.KeySets.GetAll()
+	if err != nil {
+		return err
+	}
+	for _, s := range sets {
+		s := FKeySet{KeySet: s.KeySet}
+		utils.ZeroOutTimestamps(&s)
```

Comment:
We dont want to zero out ID for keysets by default?

Reply:
This was missed. I have added it now.